prisma/seed/seedSongs.ts

```
@@ -0,0 +1,47 @@
+import { PrismaClient, Prisma, Band, Song } from "@prisma/client";
+
+export const seedSongs = async (prisma: PrismaClient, bands: Band[]): Promise<Song[]> => {
```

ESLint

[error] 'bands' is defined but never used.

(no-unused-vars)